test/components/view-toggle/view-toggle.vdiff.js

```diff
@@ -0,0 +1,41 @@
+import '../../../src/components/view-toggle/view-toggle.js';
+import { expect, fixture, html } from '@brightspace-ui/testing';
+
+const basicFixture = html`<d2l-labs-view-toggle
+toggle-options='[{"text":"Bananas","val":"overall"},{"text":"Minions","val":"minios"},{"text":"Pyjamas","val":"subject"}]'
+text="Toggle: "></d2l-labs-view-toggle>`;
+
+const noWrapFixture = html`<d2l-labs-view-toggle
+toggle-options='[{"text":"Bananas","val":"overall"},{"text":"Minions","val":"minios"},{"text":"Pyjamas","val":"subject"},{"text":"Elephant","val":"longmemory"},{"text":"Tiger","val":"fast"}]'
+text="Toggle: "></d2l-labs-view-toggle>`;
+
+const viewports = [
```

Comment:
Noticed a lot of labs tests just pick a mobile and a desktop viewport, but added in the viewports we use in our [own code](https://github.com/BrightspaceUI/creator-plus-authoring-tools/blob/main/test/utilities/constants.js#L6) for some extra coverage.

Reply:
Seeing as the component no longer has any responsive styles, I honestly wouldn't test any of these as they're just taking up time & space. I'd keep your "should not wrap" test on a smaller viewport just to assert that things don't wrap, but leave out the rest of the viewport tests.